src/main.rs

```diff
@@ -5,6 +5,22 @@
 // Full license information available in the project LICENSE file.
 //
 
+#[macro_use] extern crate log;
```

Comment:
Using the `#[macro_use]` tag here means you can just use the various `error!()`, `warn!()`, `info!()`, `debug!()` `trace!()` macros at will throughout your project without needing to explicitly `use` them in each file you want to perform logging in.